.sisyphus/drafts/layout-engine-analysis.md

```
@@ -0,0 +1,247 @@
+| Approach | Status | Bugs | Complexity | Recommendation |
+|----------|--------|------|------------|----------------|
+| **Native Split** | Working | None known | High (editly call per split) | ❌ Remove |
+| **Userland Grid/Split** | Buggy (main) | #61, #62 fixed in unmerged commit | Low (prop injection) | ✅ Merge fix, keep |
+| **Yoga Layout** | Research | N/A | Medium (new dependency) | 🤔 Future consideration |
+
+---
+
+## 1. Native Split (`src/react/renderers/split.ts`)
+
+### How it works
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)